Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY go.sum ./
 RUN go mod download
 
 ADD . .
-RUN --mount=type=cache,target=/root/.cache/go-build CGO_CFLAGS="$CGO_CFLAGS" GOOS=linux go build -trimpath -ldflags "-s -X 'github.com/flashbots/mev-boost/config.Version=$VERSION'" -v -o mev-boost .
+RUN --mount=type=cache,target=/root/.cache/go-build CGO_CFLAGS="$CGO_CFLAGS" GOOS=linux go build -trimpath -ldflags "-s -X 'github.com/flashbots/mev-boost/config.Version=$VERSION' -linkmode external -extldflags '-static'" -v -o mev-boost .
```

Comment:
Needed to compile fully static binary. Otherwise getting the following error when running the Docker image:

```
Error loading shared library libresolv.so.2: No such file or directory (needed by /app/mev-boost)
```

Reply:
This seems fine to me.